test/integration/api/access_tokens_test.rb

```diff
@@ -50,6 +50,18 @@ def setup
     end
   end
 
+  test 'create accepts an expiration time' do
+    access_token = FactoryBot.create(:access_token, owner: @admin, scopes: %w[account_management])
+
+    user_id = @admin.id
+    expires_at = 1.day.from_now.utc.iso8601
+    assert_difference(AccessToken.method(:count), 1) do
+      post_request(user_id, {access_token: access_token.value}, { expires_at: })
```

Comment:
Or maybe I don't understand the test. So:
* we create a token
* we create another token with the same value as the first one with expiration
* that works

Should the test be called - two tokens can have the same value? Although I'm not sure how do we allow that. So here's where my confusion comes from.

Reply:
They are two different tokens, the first one is needed because the endpoint itself requires an `access_token` or it returns a `403`. We could also use something like `@provider.access_tokens.first.value` instead of creating a new one, but I just copied what other tests in the suite do.

What test does is request creating a new token with expiration time and assert the new token is created and in fact has expiration time correctly set to the given value.

Reply:
Got it, I misread that token stuff :facepalm: